### ISSUE TITLE
Please add "double haploid"

### ISSUE THREAD

**Issue by tuli** [pato-ontology/pato#259]
Hello,

I need to curate a double haploid (DH) pear.

Looking on https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0001374
I do not see double haploid.

Is it because they artificially doubled?

Thank you,

Mary Ann Tuli

**Comment by ValWood** [comment]
Hi Mary Ann, wouldn't that be out of scope for PATO "double haploid" is a genotype not a phenotype?

**Comment by tuli** [comment]
Hi Val, I have to admit I hadn't thought of that - I just had a look at the
existing ploidy terms in PATO (which is what we have used in GigaDB for
some time). So, wondering why PATO includes any ploidy terms at all, I note
that PATO stands for Phenotype and Trait ontology. Could ploidy be regarded
as a trait then? I'm not sure because to me the genotype is responsible for
a trait and the phenotype is the characteristics of that trait.



*Mary Ann Tuli*

Data Editor, GigaDB
GigaScience Journal, BGI-Hong Kong
Email: maryann@gigasciencejournal.com
ORCID: 0000-0002-4667-9528 <http://orcid.org/0000-0002-4667-9528>
Website: www.gigadb.org <http://gigadb.org/>

**Comment by mellybelly** [comment]
I think this should be in GENO, @mbrush

**Comment by tuli** [comment]
Is double haploid different to all the old ploidy terms? Why are they in
PATO?

I am not familiar with GENO. I've found
http://www.ontobee.org/ontology/GENO but
searching for e.g. haploid or diploid gives no hits.

*Mary Ann Tuli*

Data Editor, GigaDB
GigaScience Journal, BGI-Hong Kong
Email: maryann@gigasciencejournal.com
ORCID: 0000-0002-4667-9528 <http://orcid.org/0000-0002-4667-9528>
Website: www.gigadb.org <http://gigadb.org/>

**Comment by mellybelly** [comment]
Only that such terms belong in GENO rather than in PATO, as they are genome features rather than phenotype/qualities. Do feel free to make requests in the GENO tracker.

**Comment by only1chunts** [comment]
Hi Val  et al. 
(Chris from GigaScience here),
Personally I think ploidy is a phenotype, its a physical chromosomal feature that can be seen/observed with chromosome spreads in a microscope and its not a gene specific thing like most genotypes. 
The fact the PATO already has a large number of ploidy terms defined would suggest that someone there has previously thought the same way! To keep the ploidy ontology terms all together would seem appropriate to me.
If all of the ploidy terms are going to be depreciated in favour of being in GENO then there is no reason why this one cant be added now to PATO and moved at the same time as the rest to GENO.
Cheers
Chris

**Comment by ValWood** [comment]
Yes I was thinking about this  more this morning!  I guess it is observable... although ploidy doesn't seem to be referred to as a phenotype...so I don't know if there is a special convention here. 

Melissa probably knows better....

**Comment by tuli** [comment]
Can we find out the history of the existing ploidy terms in PATO? Who added them?

**Comment by mellybelly** [comment]
heh, it may have been me ;-). I do think they should be ceded to GENO, which did not exist back then. We do have somewhat of a fuzzy line to draw regarding what constitutes a phenotypic quality once one gets to the molecular level. I do think that given the basic premise of G+E=P, that generally we can consider anything relating to genome attributes should not be in PATO but rather SO or GENO, but this is just my opinion.

@mbrush can you comment?

**Comment by only1chunts** [comment]
So given the ploidy stuff are already there in PATO can we get double haploid added ASAP, then worry about moving them all over to GENO in the future? We'd like to actually use the ontology term in a data entry now, otherwise it will just go into our database as free text.
Thanks Chris.

**Comment by mbrush** [comment]
Hi all - my thoughts on 'ploidy' terms as qualities vs genotypes vs phenotypes.

In PATO, the ploidy terms are framed as qualities that inhere in a genome (or an organism by extension).  I think it is perfectly valid to consider these as qualities, but this in my mind is different than being a phenotype.  Phenotypes are a particular type of quality that result from an interaction between genetics and environment, and are typically qualities observed 'downstream' of genetic sequence. Terms in PATO represent more general qualities that can be used to compose a specific phenotype in a Phenotype ontology such as HP or MP. For example, the phenotype 'cardiomegaly' (enlarged heart) is composed of the PATO term 'increased size' inhering in the Uberon anatomical term 'heart'. 
 
So, I think it is fine to leave the existing ploidy terms in PATO as qualities, and add 'double haploid' to the list.  W.r.t. whether such PATO ploidy terms should be the basis of phenotype terms or genotype terms in other ontologies (e.g. HPO or GENO), I lean toward the latter. For example, a 'double haploid genome' class would represent a 'double haploid' quality inhering in a genome - and such a class would belong in an ontology like GENO or SO, not a phenotype ontology like HPO.  This is because I tend to make the G vs P divide between things that represent the state of genetic sequence in a cell (genotype), and observable states or features 'downstream' of such sequence alterations (phenotype) - which can range from molecular changes in gene expression or protein function, all the way through gross anatomical defects and behavioral abnormalities.  But I do concede, to @only1chunts point, the notion of a double haploid genotype is in a grey area between things that clearly fall into the G camp (SNVs, insertions, deletions, translocations), and things that clearly fall into the P camp (protein malfunction, gross anatomical defects, behavioral anomalies).  From some perspectives, there may be a valid argument that an observed double haploid genome could be considered a P.
 
In summary (apologies for being long winded), I think it is fine to add 'double ploidy' as a quality to PATO.  If this is sufficient for the use case motivating this ticket, we can leave it at that.  If there are use cases that would benefit from ploidy-based genome/genotype terms, we can add them easily and quickly to GENO.  Just my 2c.

**Comment by cmungall** [comment]
Thanks @mbrush

Let's do it. I assigned to Nicole. But this might be a nice training example if anyone from GIGA wants to make a PR...

**Comment by tuli** [comment]
Thanks @mbrush and @cmungall 
I'm happy to add the new term myself. Please can you point me to the instructions on how to do so?

**Comment by nicolevasilevsky** [comment]
@tuli there are instructions here on how to do a pull request (this is for GO, but can be applied to PATO):
https://go-ontology.readthedocs.io/en/latest/DailyWorkflow.html

I'd be happy to have a call and go through this workflow together, if you like.

**Comment by matentzn** [comment]
Note the term is renamed to `doubled haploid` as per @leechuck suggestion, but has `double haploid` and `double-haploid` as synonyms.
